chemicaltoolbox/chemfp/mol2fps.xml

```diff
@@ -128,54 +127,54 @@
         <!-- FP2 -->
         <test>
             <param name="infile" value="CID_2244.sdf" ftype="sdf" />
-            <param name="fp_opts.fp_opts_selector" value="--FP2" />
+            <param name="fp_opts_selector" value="--FP2" />
```

Comment:
That will work, the more explicit way to do this would be:

https://github.com/galaxyproject/tools-iuc/blob/57493fdc88e6128bed127993464f2a6480e2774d/tools/enasearch/enasearch_search_data.xml#L85